### ISSUE TITLE
Add term delayed toe phalanx ossification

### ISSUE THREAD

**Issue by lisannesw** [obophenotype/human-phenotype-ontology#10501]
**Preferred term label:**
Delayed toe phalanx ossification

**Synonyms**
Delayed ossification involving phalangeal bones of the foot, Delayed ossification of phalanges of the foot.  

**Definition (free text, please give PubMed ID)**
Delayed maturation and calcification of any of the toe phalanges. PMC2012917,  Moore KL, Agur AMR, Dalley AF. Clinically oriented anatomy. LWW. ISBN:1451119453.

**Parent term (use hpo.jax.org/app)**
Abnormal foot bone ossification HP:0010675

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
For example, atelosteogenesis type 1 (omim10872), PMID23401428
Fibrodysplasia Ossificans Progressiva, omim135100, PMID33364240

**Your nano-attribution (ORCID)**
Andreas Zankl: 0000-0001-8612-1062, @azankl
Lisanne Wit: 0009-0007-1020-5412

**Comment by pnrobinson** [comment]
@azankl

Adding new term: 
- ID: HP:6000952
- label: Delayed toe phalanx ossification
- parents: [Abnormal foot bone ossification (HP:0010675)](https://hpo.jax.org/app/browse/term/HP:0010675)
- pmid: PMID:23401428|PMID:33364240
- def: Delayed maturation and calcification of any of the toe phalanges.
- comment: 
- orcid: orcid:0000-0001-8612-1062
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10501
- synonyms:
